src/analytics.ts

```diff
@@ -185,8 +205,15 @@ export const logEvent = (event: TrackingEvent): Promise<void> => {
         params = getLegacyAnalyticsEventParams(event as LegacyAnalyticsEvent);
         name = event.category;
     } else {
-        params = sanitizeParams(params);
-        name = sanitize(name);
+        if (Object.keys(params).length > EVENT_PARAMS_LIMIT) {
+            console.warn(
+                `Trying to log FirebaseEvent with name ${name} exceeding the limit of ${EVENT_PARAMS_LIMIT} params`,
+            );
+        }
```

Comment:
why a log in this case, and a slice in the others?

Reply:
hmmm... to be fair, I don't know what does Firebase when receive an event with more params than expected